### PR TITLE
Add no events handling

### PR DESCRIPTION
**Pull request by YingboMa**
I am not sure where should I add the test for this.

@ChrisRackauckas I made this PR, because I am not aware of a way to solve the ODE system without having an event.

### CODE REVIEW

File: src/simulate_methods/diffeqs.jl [hunk 1]
```diff
@@ -54,6 +54,7 @@ function build_pkpd_problem(_prob::DiffEqBase.AbstractJumpProblem,set_parameters
 end
 
 function ith_subject_cb(p,datai::Subject,u0,t0,ProbType,saveat,save_discont,continuity)
+  isempty(datai.events) && return t0, nothing
```
Comment (ChrisRackauckas) [inline src/simulate_methods/diffeqs.jl]: does `t0` need to be in tstops?

Comment (YingboMa) [reply src/simulate_methods/diffeqs.jl]: I am not sure what the default should be, so I used `t0`, hoping that it will have minimal runtime impact.

**Comment by YingboMa** [comment]
@ChrisRackauckas where should I add the test for this?

**Comment by ChrisRackauckas** [comment]
https://github.com/PumasAI/Pumas.jl/blob/master/test/core/multiresponses.jl#L32-L100 with a version without events.

**Pull request merged**